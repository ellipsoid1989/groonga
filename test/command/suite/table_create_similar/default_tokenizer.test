table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer "TokenNgram('n', 3)"

table_create_similar TermsSimilar Terms

dump
