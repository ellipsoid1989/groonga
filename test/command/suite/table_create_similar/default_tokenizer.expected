table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer "TokenNgram('n', 3)"
[[0,0.0,0.0],true]
table_create_similar TermsSimilar Terms
[[0,0.0,0.0],true]
dump
table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer "TokenNgram(\"n\", 3)"

table_create TermsSimilar TABLE_PAT_KEY ShortText --default_tokenizer "TokenNgram(\"n\", 3)"
